### DISCUSSION TITLE
無我的境界

### DISCUSSION THREAD

**Discussion by KozakHou** [1101-GS4719-C/Python-Community-GS4719-C#1383] [General]
對於無我的境界，我們不能不去想，卻也不能走火入魔。問題的關鍵看似不明確，但想必在諸位心中已有了明確的答案。其實，若思緒夠清晰，那麼無我的境界也就不那麼複雜了。我們都知道，只要有意義，那麼就必須慎重考慮。做好無我的境界這件事，可以說已經成為了全民運動。巴士卡里雅說過一句很有意思的話，什麼才是最本質的呢？我以為，不管我現在怎樣，我都要敢於生活，敢於用雙手擁抱現實，這才是最根本的不要為過去哭泣了，過去的已經過去了！ 這是撼動人心的。別林斯基講過一段耐人尋思的話，陰險的友誼雖然允許你得到一些微不足道的小惠，卻要剝奪掉你的珍寶－獨立思考和對真理純潔的愛！ 希望大家能從這段話中有所收穫。馬克思講過一句值得人反覆尋思的話，歷史認不那些專為公共謀福利從而自己也高尚起來的人物是偉大的。經驗證明能使大多數人得到幸福的人，他本身也是最幸福的。這似乎解答了我的疑惑。

**Comment by KozakHou** [comment]
我想要延伸補充一點

王世真在過去曾經講過，凡事力爭最好的可能性，但必須做最壞的準備。做創新的科研工作更是如此。這段話對世界的改變有著深遠的影響。一般來說，我們都有個共識，若問題很困難，那就勢必不好解決。無我到最後因何而發生？這樣看來，米丘林曾提出，我們不能等待大自然的恩賜，我們的任務就是向大自然索取。這段話看似複雜，其中的邏輯思路卻清晰可見。

**Reply by Elaine2899** [reply]
相當喜歡這個評論，做最壞的準備、爭取最好的可能性。雖然之前從來沒有聽過這樣的話，但發現這似乎是我一直努力的方向。因為各種因素，我會一直為自己準備後路，但又一直努力想追求更好的自己，所以我曾經放棄當前很好的打工，去嘗試不明確的新工作，而原因是我認為我在舊工讀中無法擁有新的收穫，而新的工作可以提升我的許多軟實力，但同時我經過了不少評估，確定自己有未來一段時間的足夠生活費才將舊的工讀辭去，轉而尋找新工作的機會。

**Comment by a3240281370** [comment]
對我來說，無我感覺是超越自我意識和自我利益的狀態。在這種境界下，我們因此不再受到執著、渴望或自我中心的束縛，而是與整體融合。這種狀態帶來深層的內在平靜、喜悅和連結感。

**Comment by Alexgns** [comment]
無我的境界，通常指的是超越自我的狀態，不受自我執著、欲望或批判的束縛，達到心流的狀態。在這種狀態下，個人可以完全投入當下的活動，無分心和自我意識，經歷深度的連結和純粹的存在。

**Comment by bestbanana295** [comment]
我覺得說要能夠達到無我的境界是哩常困難的，這通常需要有集高的專注度和忍耐度方能達到此境界，我認識的人中也只有一個人能更如此高明，讓我佩服佩服！

**Comment by 1104030360** [comment]
我認為要進入無我的境界是需要相當的自律，並且面對任何干擾時能夠平靜解決，這看似容易實則困難。

**Comment by PaulSu0905** [comment]
我對無我的境界最有印象的就是在動畫影子籃球員裡面的球員可以透過進入無我的境界變的超強，而其中也提到如果你是考自我的意識想進入那個境界是不可能的，只有在完全投入時才能進入。

**Comment by poweichen00** [comment]
我覺得無我的境界感覺非常難達到，除了要非常投入還不能被其他外在因素所影響到。

**Comment by slps101023** [comment]
無我的境界與意義的追尋
無我並非全然忘我，而是一種更高層次的自我覺察，讓人超越個人的局限，融入更大的全體之中。這樣的境界與思緒的清晰息息相關，如你所說，只要有意義，就需要慎重考慮。無我並非逃避，而是擁抱現實的勇氣和智慧。

**Comment by slps101023** [comment]
慎思的態度與行動
無我的實踐並不簡單，需要在生活中不斷磨練。它既是內心的追求，也是具體行動的體現。無論是對個人還是整個社會來說，無我都不應是一種消極的遁世，而應是積極的參與與建設。

你的分享激發了我對這些話題的更多思考。或許，「無我」的實現不在於一蹴而就，而在於每天在思想和行動中的一點點超越。
